### DISCUSSION TITLE
Recommendation for powering Raspbery Pi?

### DISCUSSION THREAD

**Discussion by Otann** [jon-harper/OmniBox#99] [Q&A] [answered]
<img width="560" alt="Screenshot 2023-03-04 at 19 50 30" src="https://user-images.githubusercontent.com/683786/222918568-9b326474-e70c-4f7a-9c50-a123236f0e25.png">

It seems that the Pi is regularly getting underpowered. Have you ever encountered this?

I've attached two simple buck converters to the main PSU and powering both Pi and Touchscreen from them. 

I've also seen that on more advanced printers, like Voron, the Pi is powered from the separate PSU, should I do the same and if so, is there one that would fit inside the case?

**Comment by jon-harper** [comment, marked as answer]
I usually do 24V PSU -> buck converter -> Raspberry Pi USB port (not 5V in). Make sure your buck converter can handle at least 3A. [This model](https://www.amazon.com/gp/product/B01NALDSJ0) by DROK is my favorite. It doesn't have screw holes, BUT it can do 5A, is pre-set to 5V, and is easy to just zip-tie to the side of the case.

You can use a separate power supply for the Pi, though the primary use case is so that the MCU can be powered off without losing power to the Pi.
